### ISSUE TITLE
Needs to update to 2.35

### ISSUE THREAD

**Issue by rien333** [pejobo/glibc-widevine#2]
Arch currently depends on glibc 2.35. Things will break if you're on 2.33, and this package currently sits at that version.

If anyone else broke their system because of this, let me know, since the fix was pretty simple. 

Oh,  and btw, there's an AUR package of this now. Not sure if you knew about, but perhaps good to link it in the README: https://aur.archlinux.org/packages/glibc-widevine

**Comment by pejobo** [comment]
Could you post your fix here? Or provide a PR for this?

**Comment by rien333** [comment]
Whoops, I was unclear. I meant that if you already happened to have upgraded your Arch ARM system, as I had, then almost everything would be broken.

Fortunately, I still had a somewhat working shell session open. 

The way I then fixed this was to simply 1) `curl -L http://mirror.archlinuxarm.org/armv7h/core/glibc-2.35-2-armv7h.pkg.tar.xz > glibc-2.35-2-armv7h.pkg.tar.xz`, and then 2) `sudo pacman -U glibc-2.35-2-armv7h.pkg.tar.xz`. In other words, I could still download the glibc from the `core/` repo, and then manually upgrade on the basis of that download. Almost everything else about pacman was broken.  Note that I had to use `curl`, because most other download utilities (`wget`, `aria2c`) were broken.
